scripts/approvalYears.py

```diff
@@ -148,7 +151,7 @@ def parseIngred(ing, uniiPT, uniiALL, missing):
                 if len(match) == 10:
                     ingreds.append(match)
                 else:
-                    if not missing.has_key(si):
+                    if si not in missing:
```

Comment:
Typo: "si"

Reply:
Variable name is correct ... "si" ... which is an awful name, but I was using it as "substance ingredient" in this case

Reply:
👍 I won't review the merge requests at night anymore... 😅